orion/pkg/repos/migrations/000004_create_table_semesters.up.sql

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE semesters
 (
     id          int unsigned NOT NULL AUTO_INCREMENT,
-    created_at  bigint(20)   NOT NULL,
-    updated_at  bigint(20)   NOT NULL,
+    created_at  datetime     NOT NULL,
+    updated_at  datetime     NOT NULL,
     deleted_at  datetime,
     semester_id varchar(64)  NOT NULL UNIQUE,
-    title       varchar(64)  NOT NULL,
+    title       varchar(255) NOT NULL,
```

Comment:
I think a title will usually follow the format: `2019_fall` or `2025_summer`, so it shouldn't be too large. 255 characters will be overkill. The reason we want to keep this tight is because imagine for every row, we allocate 255 bytes of storage for the `title` field but only use like 12 bytes. That's a lot of wasted space and quickly racks up in databases. 

On that note, when deciding these amounts, we don't have to get crazy trying to optimize. But at the same time, we also want to give ourselves a good amount of leeway because changing this later can be painful.